src/FetchTool.js

```diff
@@ -7,42 +11,43 @@ class FetchTool {
     /**
      * Is get supported? false if the environment does not support fetch.
      * @returns {boolean} Is get supported?
+     * @deprecated Fetch is always supported now
      */
     get isGetSupported () {
-        return typeof fetch !== 'undefined';
+        return typeof scratchFetch !== 'undefined';
```

Comment:
Just personal preference, I think I would `return true;` here and `isSendSupported`. But this way does make it clearer why it is deprecated.